SwanContents/swancontents/filemanager/__init__.py

```diff
@@ -1 +1,6 @@
-from .swanfilemanager import *
\ No newline at end of file
+from .swan_eos_filemanager import SwanEosFileManager
+
+try:
+    from .swan_cs3_filemanager import SwanCs3FileManager
+except:
+    pass
```

Comment:
Why the try-catch in the first place?

Reply:
Because we don't want to install the CS3 module (which brings a couple of dependencies) if we don't strictly need to. That's why I have a try-catch: if it's installed, we can use it, if not, no harm is done to the default manager.
I don't add a log because I would have to instantiate a logger, but if you try to use it and it's not importer, I think the error should be sufficient.